docs/multicast-guide.md

```diff
@@ -141,7 +142,7 @@ Support for Windows and IPv6 will be added in the future.
 
 ### Encap mode
 
-Configuration option `multicastInterfaces` is not supported with encap mode.
+The configuration option `multicastInterfaces` is not supported with encap mode.
```

Comment:
Do not add a new line between sentences. Just wrap the lines at 80.

Reply:
The sentence is 80 characters long.